apps/sms/views/conversation/test/unit/link_helper_test.js

```diff
@@ -153,12 +153,14 @@ suite('link_helper_test.js', function() {
       test('non ascii prefix', function() {
         testURLMatch('àmozilla.org', 'mozilla.org', true);
       });
+      test('Bug 941763: Add dot as URL prefix', function() {
```

Comment:
@steveck-Chung , I think it would be better if we file a new bug for filtering all the test titles which include bug numbers. Anyways, we may proceed to change the present test title in this patch itself.